src/dmd/dsymbolsem.d

```diff
@@ -4547,6 +4559,21 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
         }
     }
 
+    void checkUnionInitializers(ref VarDeclarations fields)
```

Comment:
Ddoc?

Reply:
Seemed pretty obvious from the function name, also it should be private.